pkg/cmd/provisioning/azure/create_all.go

```diff
@@ -186,6 +187,13 @@ func NewCreateAllCmd() *cobra.Command {
 			"A resource group will be created (with name derived from the --name parameter) if an installation-resource-group-name parameter was not provided. "+
 			"Note that this resource group must be provided as the installation resource group when installing the OpenShift cluster.",
 	)
+	createAllCmd.PersistentFlags().StringVar(
+		&CreateAllOpts.NetworkResourceGroupName,
+		"network-resource-group-name",
+		"",
+		"The name of the Azure resource group for network resources like the Virtual Network and Subnets used by the cluster. "+
+			"If not provided, these resources will be placed in the installation resource group",
```

Comment:
Recommend changing this to provide more info about what this parameter does,

```suggestion
		"The name of the Azure resource group in which existing Azure Virtual Network (VNet) infrastructure has been created for cluster installation. "+
		        "Cluster operators which interact with network resources will be scoped to allow management of resources in the network resource group. "+
			"This is an optional parameter that does not need to be specified when installation will not utilize an existing VNet in the install-config.yaml.",
```